src/pages/GuestInfo.tsx

```diff
@@ -0,0 +1,169 @@
+import type { PersonalInfo } from '../store/types';
+import { Box, FormField, TextInput, DateInput, Form, Button } from 'grommet';
+import { useNavigate } from 'react-router-dom';
+import { MainLayout } from '../layouts/MainLayout';
+import Logger from '../utils/logger';
+import { useCallback, useState } from 'react';
+import axios from 'axios';
+import { MessageBox } from '../components/MessageBox';
+import { DateTime } from 'luxon';
+import { useAppDispatch, useAppState } from '../store';
+import { backend } from '../config';
+import { regexp } from '@windingtree/org.id-utils';
+
+const logger = Logger('GuestInfo');
+const defaultValue: PersonalInfo = {
+  firstname: '',
+  lastname: '',
+  birthdate: new Date(),
```

Comment:
Why??? )))
you cannot assign abstract dates for such kind of properties

Reply:
default birthdate set to null